x-pack/plugins/observability/public/components/app/section/apm/index.test.tsx

```diff
@@ -80,7 +80,7 @@ describe('APMSection', () => {
     expect(getByRole('heading')).toHaveTextContent('Services');
     expect(getByText('Show service inventory')).toBeInTheDocument();
     expect(getByText('Services 11')).toBeInTheDocument();
-    expect(getByText('Throughput 900.0 tpm')).toBeInTheDocument();
+    expect(getByText('900.0 tpm')).toBeInTheDocument();
```

Comment:
Is this an okay way to fix these tests? They started failing—I think because the svg question mark icon now exists between "Throughput" and "900.0 tpm".